common/logger/src/struct_log.rs

```diff
@@ -118,123 +62,11 @@ impl StructuredLogEntry {
         ret
     }
 
-    fn clone_without_data(&self) -> Self {
-        Self {
-            message: self.message.clone(),
-            pattern: self.pattern,
-            category: self.category,
-            name: self.name,
-            module: self.module,
-            location: self.location,
-            timestamp: self.timestamp.clone(),
-            level: self.level,
-            data: HashMap::new(),
-        }
-    }
-
-    fn to_json(&self) -> Result<Value, serde_json::Error> {
-        let value = serde_json::to_value(self);
-        if let Err(e) = &value {
-            log::error!("[Logging] Failed to serialize struct log entry: {}", e);
-        }
-        value
-    }
-
-    pub(crate) fn to_json_string(&self) -> Result<String, serde_json::Error> {
-        let json = self.to_json()?;
-
-        // If the message is too long, let's truncate the message and leave helpful info
-        let mut json_string = json.to_string();
-        if json_string.len() > MAX_LOG_LINE_SIZE {
-            log::error!(
-                "[Logging] Structured log is too long, shrinking below : {}",
-                MAX_LOG_LINE_SIZE
-            );
-
-            let mut entry = self.clone_without_data();
-            if let Some(mut message) = entry.message {
-                message.truncate(MAX_LOG_LINE_SIZE - LOG_INFO_OFFSET);
-                // Leave 128 bytes for all the other info
-                entry.message = Some(message);
-            }
-            entry = entry
-                .data(
-                    "StructLogError",
-                    format!("Message exceeded MAX_LOG_LINE_SIZE {}", MAX_LOG_LINE_SIZE),
-                )
-                .data("OriginalLength", json_string.len());
-
-            // Keep important values around
-            // TODO: We should work on decreasing log sizes, as this isn't sustainable
-            for field in FIELDS_TO_KEEP {
-                if let Some(value) = self.data.get(field) {
-                    entry = entry.json_data(field, value.clone());
-                }
-            }
```

Comment:
I removed this (and didn't add back in similar logic) since we are no longer automatically converting all format message arguments into structured logs. If we start noticing an issue we can rethink this though

Reply:
As much as I really want to get rid of this, it's not just that we're automatically converting all format message arguments into structured logs, but sometimes just what we're putting in the log.

http://logging.premainnet.testnet.libra.org/_plugin/kibana/goto/8426e1aab82c15b22e09a41b2a2adf3b
https://github.com/libra/libra/blob/master/state-synchronizer/src/coordinator.rs#L334-L338

I think we probably will need something that at least keeps track of repeated messages that are that large.  We shouldn't have to cut logs, but also we need to protect ourselves from starving out the rest of the logs.  

On the other hand, we do need an example of this log that seems to keep growing in size about bad chunks.  It was 2kb and now it's 4kb.  So, we can remove it for now.

Can you make a followup issue to investigate DoS and starvation protection in the logging system around large logs?  I want to protect ourselves from accidentally shooting ourselves in the foot because of logs.

Reply:
> I removed this (and didn't add back in similar logic) since we are no longer automatically converting all format message arguments into structured logs. If we start noticing an issue we can rethink this though

Did we remove this? Are regular logs going to structured logging still? I ask because that's the only thing that goes to kibana.

Reply:
@gregnazario https://github.com/libra/libra/issues/5901 for that.

So just to be clear: Would you like me to put this back in or are we good for now?

Reply:
@davidiw Yes the series I landed yesterday unifies our logging systems. We no longer have "text" or "structured logs" Its all just logs and they all get sent to Kibana.